frontend/assets/cardTemplate.jsxt

```diff
@@ -191,7 +191,7 @@
                 txt="Visualizza la scheda dei metadati"
                 url="http://www.geoportale.piemonte.it/geonetworkrp/srv/ita/metadata.show"
                 params={{
-                    id: 2658 || model.getValue("/wfs:FeatureCollection/wfs:member/sira:AutorizzazioneUnicaAmbientale/sira:impianto/sira:Sede/sira:codiceSira/text()"),
+                    id: 6151 || model.getValue("/wfs:FeatureCollection/wfs:member/sira:AutorizzazioneUnicaAmbientale/sira:impianto/sira:Sede/sira:codiceSira/text()"),
```

Comment:
Ho cambiato l'id rispetto alla versione precedente perchè Simona mi ha dato per l'AUA quel nuovo id legato alla scheda del metadato caricato sul geoportale, in questo modo si apre la corretta scheda metadato. La url con id 2658 era stata usata solo per il prototipo.